docs/client_reference.rst

```diff
@@ -30,8 +30,8 @@ Usage example::
 
 
 .. class:: ClientSession(*, connector=None, loop=None, request_class=None,\
-                          response_class=None, cookies=None, headers=None,\
-                          auth=None)
+                          response_class=None, ws_response_class=None,\
+                          cookies=None, headers=None, auth=None)
```

Comment:
I noticed here that in the docs it lists the default params: request_class=None, response_class=None (and now ws_response_class=None). However, in the code base these are set to the default classes. Is this an oversight? Or is it intended to be that way?